Sources/FoundationInternationalization/Formatting/Number/ICULegacyNumberFormatter.swift

```diff
@@ -123,7 +123,7 @@ internal final class ICULegacyNumberFormatter : @unchecked Sendable {
     enum NumberFormatType : Hashable, Codable {
         case number(NumberFormatStyleConfiguration.Collection)
         case percent(NumberFormatStyleConfiguration.Collection)
-        case currency(CurrencyFormatStyleConfiguration.Collection)
+        case currency(CurrencyFormatStyleConfiguration.Collection, /*currency code*/ String)
```

Comment:
```suggestion
        case currency(CurrencyFormatStyleConfiguration.Collection, currencyCode: String)
```
Swift actually allows this syntax, but not sure about the Foundation style guidelines 😄